DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Services/ToHostScalingService.cs

```diff
@@ -0,0 +1,31 @@
+using Autodesk.Revit.DB;
+using Speckle.Converters.Common;
+
+namespace Speckle.Converters.RevitShared.Services;
+
+public static class ToHostScalingService
+{
+  public static double Scale(double value, string units)
+  {
+    if (string.IsNullOrEmpty(units))
+    {
+      return value;
+    }
+
+    return UnitUtils.ConvertToInternalUnits(value, UnitsToNative(units));
+  }
+
+  private static ForgeTypeId UnitsToNative(string units)
```

Comment:
Isn't this being done elsewhere?

Reply:
I thought the same thing but I don't think so. In the contextStack, we have an IHostToSpeckleUnitConverter<THostUnit> which goes from hostUnit to string, but not string to HostUnit. Maybe we should add this to that interface instead?